pages/fundamentals/operators.html

```diff
@@ -36,20 +36,20 @@ <h3>About Some Descriptions in Operator Explanations</h3>
 <li>
 	if only one of the two operands is typed,
 	then the other (untyped) operand must be representable as
-	a value of the typed of the typed operand,

```

Comment:
Thanks for these fixes and improvements.

But here, I think "the type of" is needed.